_sass/components/popovers/popover-search.scss

```diff
@@ -0,0 +1,84 @@
+.popover-search {
+  border: 0;
+  bottom: 0;
+  box-shadow: 0 2px 6px rgba($black,.075), 0 9px 25px rgba($black,0.04);
+  height: auto;
+  overflow-x: hidden;
+  overflow-y: scroll;
+  position: absolute;
+  top: 0;
+
+  &:not(.lunr-hidden) {
+    margin: 80px 0;
+    max-width: 220px;
+    opacity: 1;
+    overflow-y: auto;
+    transform: scale3d(1, 1, 1) translate3d(-50%, 0, 0);
+    transition: all .10s ease-in-out;
+    visibility: visible;
+    width: 100%;
+
+    &:hover {
+      box-shadow: 0 3px 10px rgba($black,.1), 0 9px 25px rgba($black,0.04);
+    }
```

Comment:
What is the logic for some of these styles appearing only when it does not have .lunr-hidden? It seems like the only one that should belong under this selector is opacity and visibility.